src/Template/Bake/Template/index.ctp

```diff
@@ -16,6 +16,7 @@
 <?php
 /**
   * @var \<%= $namespace %>\View\AppView $this
+  * @var \<%= $namespace %>\Model\Entity\<%= $entityClass %>[] $<%= $pluralVar %>
```

Comment:
@dereuromark  I'd argue that using collection methods has nothing to do with accessing entity properties. While the latter may be part of the further processing of collection method results, this shouldn't invalidate the ability to do the former without having the code being marked as erroneous.